Source/Fuse.Drawing.Surface/DotNetSurface.uno

```diff
@@ -674,6 +593,22 @@ namespace Fuse.Drawing
 			colors[numberOfColorPoints - 1] = endColor;
 			offsets[numberOfColorPoints - 1] = 1.0f;
 
+			var inVec = Vector.Normalize(inEnd - inStart);
+			var inLen = Vector.Length(inEnd - inStart);
+			//safety
+			if (inLen < 0.1f) {
```

Comment:
I think this is a bit large of an epsilon here. Shouldn't the code work even if this is way smaller (like 1e-5 or something)?

Reply:
The code might start misbehaving if this gets too small as it alters the offset ranges too much.  0.1 is chosen as kind of a cut-off where anything below this you can't visually see a difference anyway (this is a measure of pixel-spread of the gradient).